crates/rpc-types-trace/src/geth/erc7562.rs

```diff
@@ -0,0 +1,331 @@
+//! Geth ERC-7562 tracer types.
+
+use alloy_primitives::{map::HashMap, Address, Bytes, B256, U256};
+use serde::{Deserialize, Serialize};
+
+use crate::geth::CallLogFrame;
+
+/// The response object for `debug_traceTransaction` with `"tracer": "erc7562Tracer"`.
+///
+/// <https://github.com/ethereum/go-ethereum/pull/31006>
+#[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Erc7562Frame {
```

Comment:
can we order the fields the same way geth does it, this will make it easier to compare outputs:

https://github.com/ethereum/go-ethereum/blob/355228b011ef9a85ebc0f21e7196f892038d49f0/eth/tracers/native/erc7562.go#L51-L51